content/applications/productivity/calendar/outlook.rst

```diff
@@ -2,37 +2,65 @@
 Outlook Calendar synchronization
 ================================
 
-Synchronizing a user's Outlook Calendar with Odoo is useful for keeping track of their tasks and
+Synchronizing a user's Outlook Calendar with Odoo is useful for keeping track of tasks and
 appointments across all related applications.
 
 .. seealso::
    - :doc:`/applications/general/users/azure`
    - :doc:`/applications/general/email_communication/azure_oauth`
 
-Register the application with Microsoft Azure
-=============================================
+Setup in Microsoft Azure
+========================
 
 To sync the Outlook Calendar with Odoo's Calendar, a Microsoft Azure account is needed. Creating an
```

Comment:
This suggestion (and other suggestions like it below) are following the "italicize software/applications" and "keep the brand unitalicized" rule (e.g. Odoo *Calendar*)

-- And, technically, since Outlook is a Microsoft software application, that's why it's *Outlook Calendar* and not Outlook *Calendar*